todo_app/templates/index.html

```diff
@@ -11,8 +11,15 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in itemList %}
+          <li>{{item.title}}</li>
+        {% endfor %}
       </ul>
+      <form action = "http://localhost:5000/items" method = "post">
```

Comment:
Similarly, this action should be relative, or use url_for

Reply:
done